autoware_control_msgs/msg/Longitudinal.msg

```diff
@@ -0,0 +1,15 @@
+# Longitudinal control message
+#
+# Positive values represent forward motion (+X)
+# Negative values represent backward motion (-X)
+
+builtin_interfaces/Time stamp
+
+# Desired vehicle speed in m/s
+float32 speed;
```

Comment:
If it could be negative, is `velocity` more appropriate?
https://www.reddit.com/r/askscience/comments/3b1hcj/is_negative_speed_possible/

```suggestion
float32 velocity;
```

Reply:
Done in https://github.com/autowarefoundation/autoware_msgs/pull/11/files#diff-7d6b0eb738ad845e34035791e883a4c03fd02e4aa79597f792146993208d3c25R11